### PR TITLE
Packaging #2

### LINKED ISSUE #2: Add setup.py for packaging
KittyPy needs packaging!

### PR DESCRIPTION
**Pull request by yadudoc**
@aswathyajith Please review. Fixes #2

### CODE REVIEW

File: kittypy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from kittypy.version import VERSION
+
+__version__ = VERSION
+__author__ = "Yadu Nand B, Aswathy Ajith"
```
Comment (aswathyajith) [inline kittypy/__init__.py:4]: Should this be author or authors?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# tests_require = parse_requirements('test-requirements.txt')
+
+setup(
+    name='kittypy',
+    version=VERSION,
+    description='Get cats from the web',
+    long_description='Download cats from the web to your disk',
+    url='https://github.com/yadudoc/kittypy',
+    author='Yadu Nand Babuji',
```
Comment (aswathyajith) [inline setup.py]: Yadu Nand B for consistency? What do we do about multiple authors?

**Review by aswathyajith — CHANGES_REQUESTED** [review]

File: kittypy/__init__.py (hunk 1 above)
Comment (yadudoc) [reply kittypy/__init__.py:4]: Python's setup script does not support a list of authors, and the only way is the current method of specifying a comma separate string. Sorry!

File: setup.py (hunk 2 above)
Comment (yadudoc) [reply setup.py]: Agreed, Fixing this now.

**Review by aswathyajith — APPROVED** [review]

**Pull request merged**